src/request.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-console */
+const http = require('http');
+
+const BASE = 'http://localhost:5700';
+
+const href = BASE + '/hello-world?toCase=CAMEL';
+
+const request = http.request(href, (res) => {
```

Comment:
The response from the request is being logged to the console. This does not meet the requirement to respond with the converted text in JSON format. You should parse the response and return it in JSON format.